bundles/designer/org.eclipse.smarthome.designer.core/src/main/java/org/eclipse/smarthome/designer/core/config/PluginProjectCreator.xtend

```diff
@@ -124,8 +125,12 @@ class PluginProjectCreator implements IProjectCreator {
 	}
 	
 	def private getImportedPackages() {
-		val actionServices = CoreActivator.actionServiceTracker.services.filter(ActionService)
-		actionServices.map[it.actionClass.package.name].toSet
+		val actionServices = CoreActivator.actionServiceTracker.services?.filter(ActionService)
+		if (actionServices!=null) {
```

Comment:
The formatter handles .java files only. Correct?
Perhaps we should add the spaces around != manually.
Yes, that is a little bit pedantic :)